test/acceptance/specs/APIGatekeeperBaseSpec.scala

```diff
@@ -28,6 +28,12 @@ class APIGatekeeperBaseSpec extends BaseSpec with SignInSugar with Matchers with
     stubFor(get(urlEqualTo("/application/fa38d130-7c8e-47d8-abc0-0374c7f73216/subscription")).willReturn(aResponse().withBody("[]").withStatus(200)))
   }
 
+  def stubBlockedApplication(application: String) = {
```

Comment:
This looks like a repeat of stubApplication. Any need for it?

Reply:
Different application id. One is for an unblocked application, the other for a blocked application.